namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-integration-api-production/resources/variables.tf

```diff
@@ -56,3 +54,7 @@ variable "github_token" {
   description = "Required by the Github Terraform provider"
   default     = ""
 }
+
+variable "hosted_zone" {
+    default = "integration-api.hmpps.service.justice.gov.uk"
```

Comment:
Please use hmpps.service.justice.gov.uk, no separate zone is needed for "integration-api"